server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -136,20 +139,20 @@ module.exports = class ReportUploadRequestHandler {
         });
         return rl;
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
I think we can change any to Error

Reply:
In TypeScript, the type of a catch clause variable must indeed be any or unknown if specified, due to the fact that any value can be thrown in JavaScript, not just Error objects.

Reply:
so if we don't want to use 'any' we have to code it like:
``` javascript
catch (e: unknown) {
        if (e instanceof Error) {
          console.log(e.stack);
          errors.push(`Failed to parse a line due to: ${e.message}`);
        } else {
          // Handle or log other kinds of thrown values if necessary
          errors.push(`Failed to parse a line due to an unexpected error.`);
        }
        ```